recipes/sipbuildtool/conanfile.py

```diff
@@ -0,0 +1,127 @@
+import os
+import sys
+
+from pathlib import Path
+from io import StringIO
+
+from jinja2 import Template
+
+from conan import ConanFile
+from conan.tools.env import VirtualRunEnv
+from conan.tools.files import files
+from conans.errors import ConanException
+
+
+class SipBuildTool(object):
```

Comment:
In Python3, this is equivalent to:
```suggestion
class SipBuildTool:
```